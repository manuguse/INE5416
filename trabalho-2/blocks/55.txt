1 1 1 12 12 19 19 24 26 30 30 30
1 1 1 12 12 12 21 24 27 27 30 30
2 1 3 12 15 15 21 24 24 27 30 31
3 3 3 3 16 16 17 24 27 27 30 31
4 4 11 13 13 17 17 24 25 31 31 31
5 4 11 11 13 17 17 25 25 28 28 31
5 4 8 8 17 17 22 22 25 28 32 33
6 8 8 8 14 14 22 22 28 28 32 33
6 9 9 14 14 20 22 22 29 28 33 33
7 10 10 10 10 20 18 18 29 29 33 33
7 7 10 10 18 18 18 23 29 29 33 34
7 7 7 10 18 18 23 23 23 23 34 34