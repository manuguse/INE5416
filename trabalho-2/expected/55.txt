5 7 2 6 5 2 1 6 1 6 7 4
4 6 1 4 3 1 2 5 3 4 5 1
1 3 5 2 1 2 1 3 4 2 3 5
2 1 3 4 2 1 7 2 5 1 2 3
1 4 2 1 3 4 6 1 3 4 6 2
2 3 1 3 2 3 2 4 2 5 3 1
1 2 5 2 5 1 6 5 1 4 2 7
2 4 3 1 2 4 2 4 6 2 1 6
1 2 1 3 1 2 1 3 5 1 5 4
5 4 5 7 6 1 5 1 3 4 3 2
4 2 3 2 7 4 2 3 1 2 1 3
3 1 6 1 6 3 4 2 5 1 2 1